op-batcher/batcher/channel_builder.go

```diff
@@ -147,11 +149,21 @@ func (c *channelBuilder) ID() derive.ChannelID {
 	return c.co.ID()
 }
 
-// InputBytes returns to total amount of input bytes added to the channel.
+// InputBytes returns the total amount of input bytes added to the channel.
 func (c *channelBuilder) InputBytes() int {
 	return c.co.InputBytes()
 }
 
+// ReadyBytes returns the amount of bytes ready in the compression pipeline to
+// output into a frame.
+func (c *channelBuilder) ReadyBytes() int {
+	return c.co.ReadyBytes()
+}
+
+func (c *channelBuilder) OutputBytes() int {
+	return c.outputBytes
+}
```

Comment:
Super nice to have this - wanted to add in my batcher prs but figured it was out of scope.

Reply:
Yes :) Added it in this PR because I needed it for metrics and the evaluation of the compression ratio.